packages/types/src/sdk/events/event.ts

```diff
@@ -194,9 +194,9 @@ export enum Event {
 // a user facing event or not.
 export const AuditedEventFriendlyName: Record<Event, string | undefined> = {
   // USER
-  [Event.USER_CREATED]: `User "{{ email }}" created`,
-  [Event.USER_UPDATED]: `User "{{ email }}" updated`,
-  [Event.USER_DELETED]: `User "{{ email }}" deleted`,
+  [Event.USER_CREATED]: `User "{{ email }}" created{{#if viaScim}} via SCIM{{/if}}`,
```

Comment:
This is quite dirty, and it requires a handlebars execution with empty objects to display the title used in the dropdown. If this needs to extend further, I would recommend changing the string for an object `{title:string, template:string}`. Otherwise it will become quite hard to follow